connectivity/FEATURE_BLE/tests/UNITTESTS/doubles/fakes/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+add_subdirectory(ble)
```

Comment:
I don't think we need the `ble` subdirectory because this is already in the BLE component.
Maybe we can also flatten files in fakes/ble/source/ into fakes/, to be consistent with other components' stubs?

Reply:
We can't move fakes/ble/source/ into fakes as this directory structure expected by actual FEATURE_BLE component

For example:

https://github.com/ARMmbed/mbed-os/blob/master/connectivity/FEATURE_BLE/source/Gap.cpp#L19